pkg/network/ebpf/c/tracer-stats.h

```diff
@@ -7,10 +7,16 @@
 
 static int read_conn_tuple(conn_tuple_t *t, struct sock *skp, u64 pid_tgid, metadata_mask_t type);
 
-static __always_inline conn_stats_ts_t* get_conn_stats(conn_tuple_t *t) {
+static __always_inline u32 get_sk_cookie(struct sock *sk) {
```

Comment:
`struct sock` does have a cookie field already `sk_cookie`. Did you look at using that?

Reply:
Nevermind, that wouldn't do what you desire since it wouldn't change values.